package.json

```diff
@@ -25,8 +25,9 @@
     "@editorjs/quote": "^2.4.0",
     "@material-ui/core": "^4.11.4",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.58",
```

Comment:
Why do we need it here?

Reply:
Peer dependency of macaw, we're going to use skeletons from lab in some future (meaning: when I'll have some time to spare).

Reply:
Peer dependency of macaw, we're going to use skeletons from lab in some future (meaning: when I'll have some time to spare).

Reply:
We wouldn't need it if we'd be using `npm@7`, but we're not 🙄